src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -304,15 +346,13 @@ private async Task DebugVerifyNoErrorsAsync(MutableConflictResolution conflictRe
             {
                 try
                 {
-                    _documentOfRenameSymbolHasBeenRenamed |= documentIdsForConflictResolution.Contains(_documentIdOfRenameSymbolDeclaration);
```

Comment:
We don't need to store a separate bool value, because `RenameSpansTracker` stores all the changed document Ids during the renaming process.